pkg/rhcos/qemu.go

```diff
@@ -0,0 +1,18 @@
+package rhcos
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
+// QCOW fetches the URL of the latest Red Hat CoreOS release.
+func QCOW(ctx context.Context, channel string) (string, error) {
```

Comment:
There were a few QCOW in the previous pi[eline (for QEMU and otherwise), although I'm not sure about the new pipeline.  Your filename here suggests this is a QEMO-specific lookup, so maybe the function name should be `QEMU`?

Reply:
Right.  We aren't building an OpenStack `qcow2` yet but that's definitely on the list.